youtube_dl/extractor/letv.py

```diff
@@ -239,3 +240,75 @@ class LetvPlaylistIE(LetvTvIE):
         },
         'playlist_mincount': 7
     }]
+
+
+class LetvCloudIE(InfoExtractor):
+    IE_DESC = 'LetvCloud'
```

Comment:
Information extractor description should be a more meaningful name. You can even use non-English names, for example ```乐视云```.